node/plugin/cmd/main.go

```diff
@@ -117,6 +124,16 @@ func pluginOps(ctx *cli.Context) {
 		}
 	}
 
+	if config.Operation == plugin.OperationListQuorums {
```

Comment:
putting this operation before `operator` is initialized since that require us to have private key. I am making minimal changes to make sure we make ecdsa optional here too.